guides/session.rst

```diff
@@ -80,42 +118,78 @@ With ``Behat\Mink\Session`` you can control your browsers cookies and headers:
     // retrieving response headers:
     print_r($session->getResponseHeaders());
 
-    // set cookie:
-    $session->setCookie('cookie name', 'value');
-
-    // get cookie:
-    echo $session->getCookie('cookie name');
-
-    // delete cookie:
-    $session->setCookie('cookie name', null);
-
 .. note::
 
     Headers handling is only supported in headless drivers, because there
     is no way browser controllers can get such information out of the browser.
 
 HTTP Authentication
-~~~~~~~~~~~~~~~~~~~
+-------------------
 
-Also, Mink session has a special method to perform HTTP Basic authentication:
+The Mink session has a special method to perform HTTP Basic authentication:
 
 .. code-block:: php
 
     $session->setBasicAuth($user, $password);
 
+The method can also be used to reset a previous authentication:
+
+.. code-block:: php
+
+    $session->setBasicAuth(false);
+
 .. note::
 
     Automatic HTTP authentication is only supported in headless drivers.
     Because HTTP authentication in browser requires manual user action, that
     can't be done remotely for browser controllers.
 
+Javascript Evaluation
+---------------------
+
+The session allows you to execute or evaluate Javascript.
+
+.. code-block:: php
+
+    // Execute JS
+    $session->evaluateScript('document.body.firstChild.innerHtml = "";');
```

Comment:
You're using `evaluateScript` method twice. I guess one call should say `executeScript` instead.